lighthouse-core/gather/computed/manifest-values.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const ComputedArtifact = require('./computed-artifact');
+const icons = require('../../lib/icons');
+
+const PWA_DISPLAY_VALUES = ['minimal-ui', 'fullscreen', 'standalone'];
+
+// Historically, Chrome recommended 12 chars as the maximum short_name length to prevent truncation.
+// See #69 for more discussion & https://developer.chrome.com/apps/manifest/name#short_name
+const SUGGESTED_SHORTNAME_LENGTH = 12;
+
+class ManifestValues extends ComputedArtifact {
+
+  get name() {
+    return 'ManifestValues';
+  }
+
+  static get validityIds() {
+    return ['hasManifest', 'hasParseableManifest'];
+  }
+
+  static get manifestChecks() {
+    return [
+      {
+        id: 'hasStartUrl',
+        userText: 'Manifest contains `start_url`',
+        toPass: manifest => !!manifest.value.start_url.value
```

Comment:
🚲 🏚  nit: feels super audit-y to phrase `hasStartUrl` as *passing* in the gatherer since all of these are just facts we're checking for, what about `validate`/`isTrue`/something else